pkg/console/subresource/configmap/configmap.go

```diff
@@ -193,3 +193,13 @@ func authServerYaml(logoutRedirect string) yaml.MapSlice {
 func consoleBaseAddr(host string) string {
 	return util.HTTPS(host)
 }
+
+// Helper function that pulls the yaml struct out of the data section of a configmap yaml
```

Comment:
Does this solve the `|` issue?

Reply:
Yes, by looping through the map[]string it forces the "value" of the key "console-config.yaml" into a properly formatted string which removes the block ('|') character.

Reply:
Nice.